common/utilities/os/hresult.h

```diff
@@ -31,14 +31,16 @@ namespace hresult {
         throw std::runtime_error( descr );                                                         \
     }
 
+#if BUILD_EASYLOGGINGPP
```

Comment:
Why is this needed? Don't we have a definition for LOG_DEBUG in case BUILD_EASYLOGGINGPP isn't defined?

Reply:
removed